.travis/before_script_test.sh

```diff
@@ -1,3 +1,10 @@
-#!/usr/bin/env bash
+#!/usr/bin/env sh
 set -ev
 
+TESTING_SCRIPTS_DIR=vendor/symfony-cmf/testing/bin
+CONSOLE=${TESTING_SCRIPTS_DIR}/console
+
+export KERNEL_CLASS="Sonata\\DoctrinePHPCRAdminBundle\\Tests\\Fixtures\\App\\Kernel"
+echo '+++ create PHPCR +++'
+${CONSOLE} doctrine:phpcr:init:dbal --drop --force -vvv
+${CONSOLE} doctrine:phpcr:repository:init -vvv
```

Comment:
The whole block gets overwritten by `dev-kit` on the next run.

This should be added [here](https://github.com/sonata-project/dev-kit/blob/master/project/.travis/before_script_test.sh.twig) with a condition on the project.

Reply:
thougt dev-kit still got it in?